### ISSUE TITLE
"Shapes" "statement templates"

### ISSUE THREAD

**Issue by kcoyle** [dcmi/dctap#108]
At the DCMI24 tutorials we found that the concepts of "shapes" and "statement templates" were difficult for many participants. We need to develop some better analogies or explanations, and Ideally add these to the primer and to any future training that we do.

**Comment by philbarker** [comment]
Yes that's true. I wrote a blog post about how I visualize statement templates and shapes: https://blogs.pjjk.net/phil/templates-and-shapes-in-dublin-core-tabular-application-profile/

**Comment by kcoyle** [comment]
I'm revising the slides we used at DC24 to talk about shapes earlier. When done I'll post a link.
